### PR TITLE
Switch from Biome to ESLint

### LINKED ISSUE #36: Switch from Biome linter to ESLint
We have already migrated our formatter to Prettier (#24) to align with the setup of the official Svelte repositories. For the same reason, the linter should also be migrated to ESLint.

If Biome adds practical support for Svelte in the future, we may consider switching back.

### PR DESCRIPTION
**Pull request by ciscorn**
Close #36 

- Switch to ESLint
- Run `typos`
- This setup does not leverage Turborepo

### CODE REVIEW

File: packages/svelte-docgen/src/doc/type.ts [hunk 1]
```diff
@@ -2,181 +2,179 @@ import type { extract } from "@svelte-docgen/extractor";
 
 import type { TypeKind } from "./kind.js";
 
-export declare namespace Doc {
```
Comment (ciscorn) [inline packages/svelte-docgen/src/doc/type.ts:5]: ESLint said “don’t use namespaces”

Comment (ciscorn) [inline packages/svelte-docgen/src/doc/type.ts:5]: ESLint says “don’t use namespaces”

File: packages/svelte-docgen/src/parser/type/constructible.test.ts [hunk 2]
```diff
@@ -530,11 +530,11 @@ describe("Constructible", () => {
 			              "alias": "Iterable",
 			              "kind": "interface",
 			              "members": Map {
-			                "__@iterator@88" => {
+			                "__@iterator@21" => {
```
Comment (xeho91) [inline packages/svelte-docgen/src/parser/type/constructible.test.ts:533]: argh 😅

File: packages/svelte-docgen/src/doc/type.ts (hunk 1 above)
Comment (xeho91) [reply packages/svelte-docgen/src/doc/type.ts:5]: That's alright, you have already implemented a workaround with:

```js
import * as Doc from "./types.ts";
```

**Review by xeho91 — APPROVED** [review]
Thank you! That's a really nice cleanup here.


I noticed you fixed typos on the way. They weren't caught with `typos`? If yes, then how come CI wasn't failing on those previously? 🤔

**Comment by ciscorn** [comment]
@xeho91
It seems that the default fail_level for reviewdog/action-typos is “none.” I’ll update the configuration accordingly. 😊

**Pull request merged**